pillar/heroku/discussions.sls

```diff
@@ -112,7 +109,7 @@ heroku:
   api_key: __vault__::secret-operations/global/heroku/api_key>data>value
   config_vars:
     AKISMET_API_KEY: __vault__::secret-operations/{{ env_data.env_name }}/{{ business_unit }}/akismet>data>api_key
```

Comment:
Since it's gonna be the same key for all envs, I would also change this to `__vault__::secret-business_unit/{{ global }}/akismet>data>api_key